<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>418</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_11">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Камеры:</string>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>23</y>
         <width>121</width>
         <height>99</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Текущая камера:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="camera_comboBox"/>
        </item>
        <item>
         <widget class="QPushButton" name="add_camera_btn">
          <property name="text">
           <string>Add camera</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_cam_btn">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>126</width>
         <height>89</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="yaw_btn">
            <property name="text">
             <string>Yaw</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="yaw_spinBox">
            <property name="minimum">
             <number>-99</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="pitch_btn">
            <property name="text">
             <string>Pitch</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="pitch_spinBox">
            <property name="minimum">
             <number>-99</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="roll_btn">
            <property name="text">
             <string>Roll</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="roll_spinBox">
            <property name="minimum">
             <number>-99</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="canvas">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>400</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>160</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>160</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Модели:</string>
      </property>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>141</width>
         <height>99</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Текущая модель:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="model_comboBox_2">
          <item>
           <property name="text">
            <string>0 - all model</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="load_model_btn">
          <property name="text">
           <string>Load model</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_model_btn">
          <property name="text">
           <string>Delete</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>240</y>
         <width>126</width>
         <height>89</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QPushButton" name="move_x_btn">
            <property name="text">
             <string>move X</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="move_x_spb">
            <property name="minimum">
             <number>-99</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QPushButton" name="move_y_btn">
            <property name="text">
             <string>move Y</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="move_y_spb">
            <property name="minimum">
             <number>-99</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QPushButton" name="move_z_btn">
            <property name="text">
             <string>move Z</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="move_z_spb">
            <property name="minimum">
             <number>-99</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>126</width>
         <height>89</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="rotate_x_btn">
            <property name="text">
             <string>rotate X</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="rotate_x_spb">
            <property name="minimum">
             <number>-99</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="rotate_y_btn">
            <property name="text">
             <string>rotate Y</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="rotate_y_spb">
            <property name="minimum">
             <number>-99</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QPushButton" name="rotate_z_btn">
            <property name="text">
             <string>rotate Z</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="rotate_z_spb">
            <property name="minimum">
             <number>-99</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>340</y>
         <width>136</width>
         <height>25</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QPushButton" name="scale_btn">
          <property name="text">
           <string>scale</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="scale_spb"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
